File: RAI/Clase2/ejercicioString.c
#include<stdio.h>

int contarCaracteres(char c[]){
    int i, cantCaracteres = 0;
    for (i = 0; c[i]!= '\0'; i++)
        cantCaracteres++;
    return cantCaracteres;
}

int main(){
    char cadena[20];
    printf("Ingrese una palabra: ");
    scanf("%s", cadena);
    printf("La cantidad de caracteres es: %d", contarCaracteres(cadena));
    return 0;
}

File: RAI/Clase6/EjercicioString.java
import java.util.Scanner;

public class EjercicioString {

    /**
     * Funcion para contar caracteres de un string
     * @param s: string a ser contado
     * @return c: cantidad de caracteres
     */
    static int contarCaracteres(String s){
        return s.length();
    }
    public static void main(String[] args) {
        Scanner lector = new Scanner(System.in);

        System.out.println("Ingrese una palabra");
        //String palabra = lector.next();
        String palabra = lector.nextLine();
        //System.out.println("palabra[0] " + palabra.toCharArray()[0]);

        int cantidadCaracteres = contarCaracteres(palabra);

        System.out.println("Palabra es " + palabra + " que tiene " + cantidadCaracteres + " caracteres.");
    }
}
